display_blocks.js

```diff
@@ -173,11 +275,12 @@ function displayError() {
 
     clear(nodeInstTitle);
     clear(nodeConfigTitle);
-
-    nodeInstTitle.appendChild(document.createTextNode(instrument));
+	//var instName = document.innerHTML(instrument)
+	//instName.class = "text-center"
+    //nodeInstTitle.appendChild(instName);
     nodeConfigTitle.appendChild(document.createTextNode("Could not connect to " + instrument + ", check IBEX server is running."));
 
-	document.getElementById("inst_name").appendChild(nodeInstTitle);
+	document.getElementById("top_bar").innerHTML = instrument;
```

Comment:
I think this is meant to display an IBEX-style "FAKE is UNKNOWN" with a yellow background, but it doesn't seem to be working like that currently:

![image](https://user-images.githubusercontent.com/24375813/29161611-d11b3654-7dae-11e7-9d51-702ca8917c5c.png)